lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -616,24 +629,24 @@ sub _create_clone {
       && $cluster_job_info->{ok};
 
     # Duplicate settings (with exceptions)
+    my %spec_settings = %$settings;
+    my %main_settings
+      = map { (delete $spec_settings{$_}) // $self->$_ } qw(TEST VERSION ARCH FLAVOR MACHINE BUILD DISTRI);
```

Comment:
We already have a list of such parameters in SCENARIO_KEYS and SCENARIO_WITH_MACHINE_KEYS in lib/OpenQA/Schema/Result/Jobs.pm . I suggest you define another constant in there for what you want to use here. I don't have a good name for that ready though

Reply:
I've been thinking the same and there's also a very similar list in `create_from_settings`. Maybe I can at least de-duplicate the common denominator.